js/src/api/subscriptions/personal.js

```diff
@@ -37,12 +39,33 @@ export default class Personal {
     ]);
   }
 
+  // FIXME: Because of the different API instances, the "wait for valid changes" approach
+  // doesn't work. Since the defaultAccount is critical to operation, we poll in exactly
+  // same way we do in ../eth (ala same as eth_blockNumber) and update. This should be moved
+  // to pub-sub as it becomes available
   _defaultAccount = () => {
+    const nextTimeout = (timeout = 1000) => {
+      setTimeout(() => {
```

Comment:
The polling gets done once for each subscription instance - which matches up with the API instances. (So while an API is loaded, it will poll.) Only part being managed is the subscriber.